src/testUtils/java/uk/gov/hmcts/reform/wataskmonitor/config/GivensBuilder.java

```diff
@@ -0,0 +1,259 @@
+import uk.gov.hmcts.reform.wataskmonitor.entities.camunda.CamundaProcessVariables;
+import uk.gov.hmcts.reform.wataskmonitor.entities.camunda.CamundaSendMessageRequest;
+import uk.gov.hmcts.reform.wataskmonitor.entities.camunda.CamundaValue;
+import uk.gov.hmcts.reform.wataskmonitor.entities.documents.Document;
+import uk.gov.hmcts.reform.wataskmonitor.services.AuthorizationHeadersProvider;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicReference;
+
+import static java.time.ZonedDateTime.now;
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.CoreMatchers.is;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static uk.gov.hmcts.reform.wataskmonitor.config.SecurityConfiguration.AUTHORIZATION;
+import static uk.gov.hmcts.reform.wataskmonitor.config.SecurityConfiguration.SERVICE_AUTHORIZATION;
+import static uk.gov.hmcts.reform.wataskmonitor.entities.camunda.CamundaMessage.CREATE_TASK_MESSAGE;
+import static uk.gov.hmcts.reform.wataskmonitor.entities.camunda.CamundaProcessVariables.ProcessVariablesBuilder.processVariables;
+import static uk.gov.hmcts.reform.wataskmonitor.entities.documents.DocumentNames.NOTICE_OF_APPEAL_PDF;
+import static uk.gov.hmcts.reform.wataskmonitor.utils.Common.CAMUNDA_DATA_TIME_FORMATTER;
+
+@Slf4j
+public class GivensBuilder {
+
+    private final RestApiActions camundaApiActions;
+    private final RestApiActions restApiActions;
+    private final AuthorizationHeadersProvider authorizationHeadersProvider;
+    private final DocumentManagementFiles documentManagementFiles;
+
+    private final CoreCaseDataApi coreCaseDataApi;
+
+    public GivensBuilder(RestApiActions camundaApiActions,
+                         RestApiActions restApiActions,
+                         AuthorizationHeadersProvider authorizationHeadersProvider,
+                         CoreCaseDataApi coreCaseDataApi,
+                         DocumentManagementFiles documentManagementFiles
+    ) {
+        this.camundaApiActions = camundaApiActions;
+        this.restApiActions = restApiActions;
+        this.authorizationHeadersProvider = authorizationHeadersProvider;
+        this.coreCaseDataApi = coreCaseDataApi;
+        this.documentManagementFiles = documentManagementFiles;
+
+    }
+
+    public String createCcdCase() {
+        Headers headers = authorizationHeadersProvider.getLawFirmAuthorization();
+        String userToken = headers.getValue(AUTHORIZATION);
+        String serviceToken = headers.getValue(SERVICE_AUTHORIZATION);
+        UserInfo userInfo = authorizationHeadersProvider.getUserInfo(userToken);
+
+        Document document = documentManagementFiles.getDocument(NOTICE_OF_APPEAL_PDF);
+
+        StartEventResponse startCase = coreCaseDataApi.startForCaseworker(
```

Comment:
Pact test for startForCaseworker. See https://github.com/hmcts/wa-task-management-api/pull/249/files as you can reuse these.